src/test_report.py

```diff
@@ -48,12 +116,18 @@ def run(self):
                             loader=jinja2.FileSystemLoader("./config/reports/")
                         )
                 template = templateEnv.get_template("test-report.jinja2")
```

Comment:
maybe we could rename `templateEnv` -> `template_env` to stick to the naming conventions